Yoast/javascript
# P2-781 Remove warning blocks when they do not apply anymore

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/schema-blocks] Removes a warning block when its removed block is re-added.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* **Note**: test this code in the WordPress SEO Premium plugin.
* Create a post.
* Add a job posting block.
* Remove a required or recommended block. E.g. the job title.
	* You should see a warning message. 
* Re-add the job title.
	* Test out different approaches:
		* Using the job posting's block inserter.
		* Using the 'Add' button next to the removed block in the sidebar of the job posting.
		* Using Gutenberg's undo button.  
* In all cases the warning message should be removed, and the removed required/recommended block re-added.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
